File: android-example-java/app/src/main/java/com/instana/android/example/ExampleApp.java
package com.instana.android.example;

import android.app.Application;

import com.instana.android.Instana;
import com.instana.android.core.InstanaConfig;

public class ExampleApp extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

        InstanaConfig instanaConfig = new InstanaConfig(
                KEY,
                REPORTING_URL
        );
        Instana.setup(this, instanaConfig);
    }
}

File: android-example-kotlin/app/src/main/java/com/instana/android/example/ExampleApp.kt
package com.instana.android.example

import android.app.Application
import com.instana.android.Instana
import com.instana.android.core.InstanaConfig

class ExampleApp : Application() {

    override fun onCreate() {
        super.onCreate()

        val instanaConfig = InstanaConfig(
            key = KEY,
            reportingURL = REPORTING_URL
        )
        Instana.setup(this, instanaConfig)
    }
}
